tests/suites/test_suite_pkparse.data

```diff
@@ -987,7 +987,7 @@ depends_on:MBEDTLS_PEM_PARSE_C:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP192R1_ENABLED
 pk_parse_keyfile_ec:"data_files/ec_prv.sec1.comp.pem":"NULL":0
 
 Parse EC Key #3 (SEC1 PEM encrypted)
-depends_on:MBEDTLS_DES_C:MBEDTLS_PEM_PARSE_C:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP192R1_ENABLED:MBEDTLS_CIPHER_MODE_CBC:MBEDTLS_HAS_MD5_VIA_LOWLEVEL_OR_PSA
+depends_on:MBEDTLS_DES_C:MBEDTLS_PEM_PARSE_C:MBEDTLS_ECP_C:MBEDTLS_ECP_DP_SECP192R1_ENABLED:MBEDTLS_CIPHER_MODE_CBC:MBEDTLS_HAS_ALG_MD5_VIA_LOWLEVEL_OR_PSA
```

Comment:
CI failed. Here the depenency on `MBEDTLS_HAS_ALG_MD5_VIA_LOWLEVEL_OR_PSA` is inconsistent with `PEM_RFC1421`:
https://github.com/Mbed-TLS/mbedtls/blob/7fdf5497ee05c87219c589edaa0d5de74e800cbb/library/pem.c#L51-L57

Reply:
[This is out of scope of 3.4.1](https://github.com/Mbed-TLS/mbedtls/pull/8022#discussion_r1282936509). I've amended the last commit to only do the DES→AES change and not fix an unrelated bug. Fixing that bug will happen in https://github.com/Mbed-TLS/mbedtls/pull/8029.